javascript/exercises/objectIntro.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Exercises: Objects</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link href="../../common/css/bootstrap.css" rel="stylesheet">
+  </head>
+ <body>
+<div class="container">
+<h2>Exercises: Objects</h2>
+
+
+<h3 class="page-header">Object Property Access</h3>
+
+<p><em>Major US Cites:  List all major cities two times, in the majorCities object, using <code>console.log</code> statements. The first time, list all the major cities using dot notation. The second time, list all the major cities using bracket notation. </em></p>
```

Comment:
I like this exercise! I think we can do more of a ramp up. What do you think of this: "Here is an object that includes major cities. Use `console.log()` to print properties of the object. Print all the properties using dot notation, then bracket notation"